### ISSUE TITLE
puffing-billy: Request to https://... failed with error: Content-decoder error

### ISSUE THREAD

**Issue by zedtux** [oesmith/puffing-billy#204]
I'm trying to figure out how to test the [Braintree payment gateway](https://www.braintreepayments.com/) (and their [DropIn UI](https://developers.braintreepayments.com/guides/drop-in/overview/javascript/v2)) in my Rails application.

So far, I have my test passing without any saved fixture files, but then when I run it again, I have the following error in logs (and my test is failing):

```bash
puffing-billy: Request to https://api.sandbox.braintreegateway.com:443/merchants/<braintree-merchant-id>/client_api/v1/configuration?authorizationFingerprint=8e2414b70f7e776c2bed9bdbe67f304f33310a29623c8a81d0a93ad18f63378a%7Ccreated_at%3D2017-11-13T10%3A32%3A31.368242957%2B0000%26customer_id%3D255624599%26merchant_id%3D<braintree-merchant-id>%26public_key%3D<braintree-public-key>&callback=callback_json4fd4f6c4ff2f4920abd99a9c85cca184 failed with error: Content-decoder error
```

I did a small project which reproduces the problem: https://github.com/zedtux/cucumber-braintree.

Can you please guide me to a solution?

**Comment by ronwsmith** [comment]
@zedtux is this still an issue after following recent SSL changes / documentation?

**Comment by zedtux** [comment]
@ronwsmith what recent SSL changes / documentation?

**Comment by ronwsmith** [comment]
@zedtux See the referenced issue above your comment ^^

**Comment by zedtux** [comment]
Thank you @ronwsmith, I will check that.

**Comment by ronwsmith** [comment]
@zedtux Is this still an issue?

**Comment by zedtux** [comment]
@ronwsmith well, this issue is stale for months, and I abandoned testing thing (repo deleted since I migrated to GitLab), so it is still an issue, but I'm not looking at it anymore.

Feel free to close this issue if no one care.

**Comment by ronwsmith** [comment]
Thanks for letting me know!  I'll close it out for now.  It's still referenced by the open PR.

**Comment by ronwsmith** [comment]
Fixed in v2.1.0
